src/Component/Locations.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import axios from 'axios'
+class Locations extends React.Component {
+    constructor(){
+        super();
+        this.state = {
+            cities:[],
+            showCity: false
+        }
+    }
+
+    ghibliCities = async () =>{
+        try{
+       const { data } = await axios.get("https://ghibliapi.herokuapp.com/locations")
+    
+       this.setState({cities:data})
+        } catch (e){
+            console.log(e)
+        }
+    }
+
+    componentDidMount(){
+        this.ghibliCities()
+    }
+    handleClick = () =>{
+        if(this.state.showCity === true){
+            this.setState({showCity:false})
+        }else {
+            this.setState({showCity:true})
+        }
+    }
+    render() {
+        const { cities, showCity } =this.state
+        let city = cities.map((ghibliaCity) => {
```

Comment:
You can be a little more general when setting parameter names. Instead of using ghibliaCity, you can just use city. Also cities.map will return an array so calling it city may cause readability issues. Functionally, this looks great!

 For example we can write it like:
`let cityElements = cities.map((city) => {`